contracts/ClaimsManager.sol

```diff
@@ -269,6 +269,48 @@ contract ClaimsManager is
         );
     }
 
+    function updatePolicy(
+        address claimant,
+        uint224 coverageAmountInUsd,
+        uint32 claimsAllowedFrom,
+        uint32 claimsAllowedUntil,
+        string calldata policy
+    ) external override returns (bytes32 policyHash) {
+        require(isAdmin(msg.sender), "Sender not admin");
```

Comment:
Why not use the [onlyAdmin](https://github.com/api3dao/claims-manager/blob/paid-policy-agent/contracts/ClaimsManager.sol#L42) modifier?

Reply:
Forgot about it :sweat_smile: